go/bind/keybase.go

```diff
@@ -4,6 +4,7 @@
 package keybase
 
 import (
+	"context"
```

Comment:
Is it intended to use `context` instead of `golang.org/x/net/context` here? I guess it wouldn't matter as long as this doesn't cross existing context chains.